src/content/docs/en/guides/imports.mdx

```diff
@@ -323,7 +324,7 @@ Other files may have various different interfaces, but `import.meta.glob()` acce
 interface CustomDataFile {
   default: Record<string, any>;
 }
-const data = await import.meta.glob<CustomDataFile>('../data/**/*.js');
+const data = import.meta.glob<CustomDataFile>('../data/**/*.js');
```

Comment:
Just checking, normally we've been removing `await` when there is eager loading!

Reply:
`await` is removed because `import.meta.glob` never returns a promise, but it returns a record of functions that return promises when called. There's no usage example, and we can't showcase this easily, but whether someone needs `eager: true` will probably come down to what they're importing and why. The Vite docs cover what eager does, and we link to that above.